src/config/config.ts

```diff
@@ -478,6 +495,25 @@ export class Config implements IConfig {
   protected get isProd() {
     return isProd()
   }
+
+  /**
+   * npm dependency name (key of the map) can be an alias ("foo": "npm:some-module-name@0.0.0")
+   * this function takes a list of command plugins instances and then matches those against
+   * the resolved set of npm dependency names
+   * @private
+   * @returns {Alias[]}
+   */
+  private resolvePluginAliasNames(): Alias[] {
```

Comment:
Might want a better name for the function.
Creates a map of alias to actual npm module name

Reply:
I would prefer for this information to be added to the `Plugin` class so that we can use it elsewhere if we need to, .e.g

```
plugin.pluginAlias: '@sf/env'
```

Reply:
I'll have a look